docs/current/sailbot_workspace/how_to.md

```diff
@@ -134,7 +134,49 @@ Each application runs in a Docker container. Containers can be managed using Doc
         docker stop <container>
         ```
 
-## Temporarily add apt packages
+## Manage software packages
+
+### Add apt or python dependencies to ROS packages
+
+If running your ROS packages requires external dependencies from an apt repository or python package, the following
+should be added to the `package.xml` file in the root directory of the ROS package:
+
+```xml
+<exec_depend>ROSDEP_KEY</exec_depend>
+```
+
+Replace `ROSDEP_KEY` with the rosdep key for the dependency, which can be found online.
+
+=== ":material-debian: Apt Dependencies"
+    - Rosdep keys for apt repositories can be found [here](https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml){target=_blank}
+    - Use the key associated with **debian** since sailbot workspace uses Ubuntu, a Linux distribution based on debian
+    - Do not include the square brackets in `package.xml`
+
+=== ":material-language-python: Python Dependencies"
+    - Rosdep keys for python packages can be found [here](https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml){target=_blank}
+    - Use the key associated with **debian** since sailbot workspace uses Ubuntu, a Linux distribution based on debian
+    - Make sure that the key you are using is for **Python 3** and not Python 2
```

Comment:
Maybe we could clarify this by just saying you need to change the key from `python-pyproj` to `python3-pyproj` for the installation to work?

Reply:
That's a good point, I'm not sure if they all follow that convention. For sure, I'll change it to say to add the 3 and if it doesn't work then try removing it. I have a feeling adding the 3 will work more often than not.